sklearn/covariance/_empirical_covariance.py

```diff
@@ -103,10 +103,8 @@ def empirical_covariance(X, *, assume_centered=False):
             "Only one sample available. You may want to reshape your data array"
         )
 
-    if assume_centered:
-        covariance = np.dot(X.T, X) / X.shape[0]
-    else:
-        covariance = np.cov(X.T, bias=1)
+    ddof = 0 if assume_centered else 1
+    covariance = (X.T @ X) / (X.shape[0] - ddof)
```

Comment:
Actually, I forgot an important step that is done in `np.cov` that is the centering :)
It explains the time difference in benchmark as well.

So we should only modify the second line `np.cov(X.T, ddof=1)`